src/verifications/verifications.ts

```diff
@@ -1319,15 +1350,15 @@ export class Verifications extends Logger {
    * @return     {Promise<string>}  verificationId
    */
   public async setVerification(
-      issuer: string,
-      subject: string,
-      topic: string,
-      expirationDate = 0,
-      verificationValue?: any,
-      descriptionDomain?: string,
-      disabelSubVerifications = false,
-      isIdentity = false,
-    ): Promise<string> {
+    issuer: string,
+    subject: string,
+    topic: string,
+    expirationDate = 0,
+    verificationValue?: any,
+    descriptionDomain?: string,
+    disabelSubVerifications = false,
```

Comment:
maybe we can change the "disabelSubVerifications" to the correct phrase "disableSubVerifications"

Reply:
fixed